app/data_model/session_store.py

```diff
@@ -69,8 +72,10 @@ def _load(self):
 
         if self._eq_session:
             self.user_id = self._eq_session.user_id
+
             if self._eq_session.session_data:
-                self.session_data = json.loads(self._eq_session.session_data, object_hook=lambda d: SessionData(**d))
+                session_data = StorageEncryption(self.user_id, self.user_ik).decrypt_data(self._eq_session.session_data)
```

Comment:
This assumes that the session is encrypted, which isn't the case if someone already has an active session when we deploy this. We should be able to make it work with un-encrypted sessions on load, which will then be encrypted on save.